config/initializers/active_job.rb

```diff
@@ -1,5 +1,11 @@
 ActiveJob::Base.queue_adapter = if Rails.env.test?
                                   :test
+                                elsif Rails.env.development? || Figaro.env.pt_mode == 'on'
+                                  # letter_opener_web (used during development only)
+                                  # doesn't currently work with sidekiq running in a separate
+                                  # container since it relies on a shared ./tmp directory.
+                                  # https://github.com/fgrehm/letter_opener_web/issues/35
+                                  :inline
                                 else
```

Comment:
Ideally, the development environment should mimic the production environment as closely as possible. If Sidekiq never runs in development, then we won't know if there are any issues with it. Instead of trying to work around `letter_opener`, what do you think about exploring alternatives that don't write to  a `tmp` directory, such as [Mailcatcher](https://mailcatcher.me/), or this [custom solution](https://gist.github.com/scmx/8761d6a12f63d918b33e)?